src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/Adjournment.java

```diff
@@ -97,13 +98,16 @@ public class Adjournment {
     @JsonProperty("adjournmentInProgress")
     private YesNo adjournmentInProgress;
 
+    @SuppressWarnings("unused")
     @JsonIgnore
     public List<JudicialUserBase> getPanelMembers() {
         List<JudicialUserBase> panelMembers = Arrays.asList(this.panelMember1,
             this.panelMember2,
             this.panelMember3);
 
-        return panelMembers.stream().filter(Objects::nonNull).collect(Collectors.toList());
+        return panelMembers.stream().filter(Objects::nonNull)
+            .filter(panelMember -> nonNull(panelMember.getIdamId()) || nonNull(panelMember.getPersonalCode()))
```

Comment:
Under what circumstances would a panel member saved in Adjournment object have neither idamId nor personalCode? If that happened, why would we want to leave them behind here? Were they not on the case? How did they make it that far without either of those properties?

Reply:
So for this function all I want is for it to return a panel member if there is something there then where it is called they can do those checks if they want. In reality this wont happen thanks to the typeahead functionality. It also makes it easier to test

Reply:
Ok makes sense